packages/tupaia-web/src/features/Map/MarkerLayer/processMeasureData.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { calculateRadiusScaleFactor, getMeasureDisplayInfo } from '@tupaia/ui-map-components';
+import { Entity } from '@tupaia/types';
+
+interface processMeasureData {
+  measureData: any;
+  entitiesData: any;
+  serieses: any;
+  hiddenValues: any;
+}
```

Comment:
I wonder if we should add types for these?

Reply:
yes agreed. I'm just working on exporting the types that I need out of ui-map-components